bin/update-version.php

```diff
@@ -14,8 +14,8 @@
 	if ( stripos( $line, ' * Version: ' ) !== false ) {
 		$line = " * Version: {$package->version}\n";
 	}
-	if ( stripos( $line, "\tdefine( 'WC_ADMIN_VERSION_NUMBER'," ) !== false ) {
-		$line = "\tdefine( 'WC_ADMIN_VERSION_NUMBER', '{$package->version}' );\n";
+	if ( stripos( $line, ">define( 'WC_ADMIN_VERSION_NUMBER'," ) !== false ) {
+		$line = "\t\t\$this->define( 'WC_ADMIN_VERSION_NUMBER', '{$package->version}' );\n";
```

Comment:
Can we future proof this a bit more by manipulating a constant on the class? Or even just adding a comment above this line alerting anyone who refactors this code about the automation could be sufficient. This was the source of the issue because an extra tab was added during a refactor.

Reply:
I went the comment route in 449c274 since the version constant is used in a few places outside of the class: [#](https://github.com/woocommerce/wc-admin/blob/b493cc1c2ef51965084685703755ec7521f0efa5/includes/class-wc-admin-install.php#L45), [#](https://github.com/woocommerce/wc-admin/blob/d8809418930c1ad279b5fd5aaa873e654256dc09/includes/class-wc-admin-loader.php#L113).